scenarios.js

```diff
@@ -1,438 +1,442 @@
+//special_rules should be treated with some kind of macro that recognises them and applies them when loading

```

Comment:
This remains a TODO, right? Maybe want to track this in an issue instead of a comment here.